pkgs/applications/misc/solaar/default.nix

```diff
@@ -47,6 +54,17 @@ python3Packages.buildPythonApplication rec {
     install -Dm444 -t $udev/etc/udev/rules.d rules.d-uinput/*.rules
   '';
 
+  dontWrapGApps = true;
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  # no tests
+  doCheck = false;
```

Comment:
Doesn't it simply do nothing if there are no tests?

Reply:
yes but we document it explicitly otherwise it is unclear in review